test/fixtures/schema/array-of-arrays.json

```diff
@@ -0,0 +1,153 @@
+                                        },
+                                        "value": {
+                                          "element": "array",
+                                          "content": [
+                                            {
+                                              "element": "array",
+                                              "content": [
+                                                {
+                                                  "element": "number",
+                                                  "content": 1
+                                                },
+                                                {
+                                                  "element": "number",
+                                                  "content": 2
+                                                },
+                                                {
+                                                  "element": "number",
+                                                  "content": 3
+                                                },
+                                                {
+                                                  "element": "number",
+                                                  "content": 4
+                                                }
+                                              ]
+                                            }
+                                          ]
+                                        }
+                                      }
+                                    }
+                                  ]
+                                }
+                              ]
+                            },
+                            {
+                              "element": "asset",
+                              "meta": {
+                                "classes": [
+                                  "messageBody"
+                                ]
+                              },
+                              "attributes": {
+                                "contentType": "application/json"
+                              },
+                              "content": "{\n  \"tags\": [\n    [\n      1,\n      2,\n      3,\n      4\n    ]\n  ]\n}"
+                            },
+                            {
+                              "element": "asset",
+                              "meta": {
+                                "classes": [
+                                  "messageBodySchema"
+                                ]
+                              },
+                              "attributes": {
+                                "contentType": "application/json"
+                              },
+                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"properties\": {\n    \"tags\": {\n      \"type\": \"array\"\n    }\n  }\n}"
```

Comment:
Where is the second array which was nested in the first array? I know, we don't have it in boutique, but we were planning to fix it in drafter.